.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: Report an issue about using Custodian
+labels: [kind/bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what happened!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: expected-bahavior
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what you expected to happen.
+      placeholder: What were you expecting to happen?
+      value: "It did this instead of this"
```

Comment:
`"It did this instead of this"` --> `"I expected 2 + 2 to equal 4, but instead 2 + 2 equaled 6!"`

Same reasons as above